_components/header/header.html

```diff
@@ -92,85 +86,52 @@ <h2>Header usability</h2>
 <h2 class="usa-heading heading-margin-alt">Header accessibility</h2>
 <ul class="usa-content-list">
   <li>
-    Include skip navigation links to allow users with screen readers to bypass
-    long navigation lists. Make sure you include an id at the beginning of your
-    main content and that it matches the skipnav link. Find more <a href="http://webaim.org/techniques/skipnav/">information on these links</a> at WebAIM.
-
+    <strong>Add a skip navigation link before the header.</strong> Include skip navigation links to allow users with
+    screen readers to bypass long navigation lists. Make sure you include an <code>id</code> at the beginning of your
+    main content and that it matches the skip navigation link. Find more <a
+      href="http://webaim.org/techniques/skipnav/">information on these links</a> at WebAIM.
```

Comment:
Wondering if there is documentation on `usa-skipnav` that we can point users to.